doc/source/conf.py

```diff
@@ -19,11 +19,16 @@
 
 import cloud_sptheme as csp
 
+import regolith
 from regolith import __version__ as REGOLITH_VERSION
 from regolith.fsclient import _id_key, dump_json, json_to_yaml
 from regolith.main import CONNECTED_COMMANDS, DISCONNECTED_COMMANDS
 from regolith.schemas import EXEMPLARS, SCHEMAS
 
+autodoc_mock_imports = [
```

Comment:
please can you double check that this is needed?

Reply:
Yes, otherwise it would run into issues like e.g. WARNING: autodoc: failed to import module '...' from module 'regolith'; the following exception was raised: No module named 'regolith'. Something like that. It is the solution that resolves this issue that gives in the api workflow.